server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/VariableUsageNode.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree;
+
+import org.eclipse.lsp.cobol.core.CobolParser;
+import org.eclipse.lsp.cobol.core.model.Locality;
+
+/**
+ * The class represents variable usage in COBOL program.
+ * This must be extended with a link to variable definition.
+ */
+public class VariableUsageNode extends Node {
```

Comment:
Why doesn't it have a `@Value` annotation?

Reply:
No idea. I'll fix it.